server/src/main/java/org/elasticsearch/index/engine/CombinedDeletionPolicy.java

```diff
@@ -296,10 +317,20 @@ private static Set<String> listOfNewFileNames(IndexCommit previous, IndexCommit
     }
 
     /**
-     * Checks whether the deletion policy is holding on to acquired index commits
+     * Checks whether the deletion policy is holding on to externally acquired index commits
      */
     synchronized boolean hasAcquiredIndexCommits() {
```

Comment:
Can we rename this method and its counterpart in `InternalEngine` to `hasAcquiredIndexCommitsForTesting()` or something similar? Also, add javadoc to use in tests only.

Reply:
Good idea, I've renamed it.